tests/common/ixia/ixia_fixtures.py

```diff
@@ -0,0 +1,141 @@
+"""This module contains the necessary fixtures for running test cases with
+ Ixia devices and IxNetwork. If more fixtures are required, they should be 
+included in this file.
+"""
+
+import pytest
+import pprint
+from common.devices import FanoutHost
+from ixnetwork_restpy import SessionAssistant, Files
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_ip(testbed):
+    """ 
+    In an Ixia testbed, there is no PTF docker. 
+    Hence, we use ptf_ip field to store Ixia API server. 
+    This fixture returns the IP address of the Ixia API server.
+
+    Args: 
+       testbed (pytest fixture): The testbed fixture.
+
+    Returns:
+        Ixia PTF server (API server) IP
+    """
+    return testbed['ptf_ip']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_user(duthost):
+    """
+    Return the username of Ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
+        Ixia PTF server (API server) username.
+    """
+    return duthost.host.options['variable_manager']._hostvars[duthost.hostname]['secret_group_vars']['ixia_api_server']['user']
+
+
+@pytest.fixture(scope = "module")
+def ixia_api_serv_passwd(duthost):
+    """
+    Return the password of Ixia API server.
+
+    Args:
+        duthost (pytest fixture): The duthost fixture.
+ 
+    Returns:
```

Comment:
Returns -> Return

Reply:
Google style docstring recommends “Returns” instead of “Return”(https://google.github.io/styleguide/pyguide.html). So, I kept it as it is.